pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -335,24 +335,52 @@ func (a v1alpha1PolicyApplier) InboundFilterChain(sdsUdsPath string, meta *model
 	if a.policy == nil || len(a.policy.Peers) == 0 {
```

Comment:
just change the method signature to accept the model.Proxy struct and not just the node metadata.

Reply:
Done